dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -13,226 +13,412 @@ The Sliceviewer in Workbench has the joint functionality of the SpectrumViewer a
+Test the :ref`Nonorthogonal view<mantid:sliceviewer_nonortho>`
 
 .. figure:: ../../../../docs/source/images/wb-sliceviewer51-nonorthobutton.png
    :class: screenshot
    :align: center
 
-B. This tests that the sliceviewer gets the correct basis vectors for an ``MDHisto`` object from a non-axis aligned cut.
+1. Click the nonorthogonal view button in the toolbar
+
+    - This should disable ROI and lineplot buttons in the toolbar
+    - This should automatically turn on gridlines
+    - When H and K are the viewing axes the gridlines should not be perpendicular to each other
+    - The features in the data should align with the grid lines
+
+2. Zoom and pan
+
+    - Confirm the autoscaling of the colorbar works in non-orthogonal view
+
+3. Change one of the viewing axes to be `L` (e.g. click `X` button next to L in top left of window)
+
+    - Gridlines should now appear to be orthogonal
+
+4. For ``md_4D`` only change one of the viewing axes to be `E` (e.g. click `Y` button next to `E` in top left of window)
+
+    - Nonorthogonal view should be disabled (only enabled for momentum axes)
+    - Line plots and ROI should be enabled
+    - Change the viewing axis presently selected as `E` to be a momentum axis (e.g. `H`)
+
+            - The nonorthogonal view should be automatically re-enabled.
+
+Test the :ref`Peak Overlay<mantid:sliceviewer_peaks_overlay>`
+
+.. figure:: ../../../../docs/source/images/wb-sliceviewer51-peaksbutton.png
+   :class: screenshot
+   :align: center
+
+1. Click to peak overlay button in the toolbar
+2. Check the `Overlay?` box next to ``peaks``
+
+    - This should open a table (peak viewer) on the RHS of the sliceviewer window - it should have a two rows corresponding to peaks at HKL = (1,0,1) and (1,0,0).
+
+3. Double click a row
+
+    - It should change the slicepoint along the integrated momentum axis and zoom into the peak - e.g. in (X,Y) = (H,K) then the slicepoint along L will be set to 1 and there will be a cross at (0,0).
+    - Note for ``md_4D`` the cross should be plotted at all E (obviously a Bragg peak will only be on the elastic line but the peak object has no elastic/inelastic logic and the sliceviewer only knows that `E` is not a momentum axis, it could be temperature etc.).
+
+4. Click Add Peaks in the Peak Actions section at the top of the peak viewer
+5. Click somewhere in the colorfill plot
+
+    - Confirm a peak has been added to the table at the position you clicked
+
+6. Click Remove Peaks
+7. Click on the cross corresponding to the peak you just added
+
+    - Confirm the correct row has been removed from the table
+    - The cross should be removed from the plot
+
+8. Repeat the above steps 1-7 in non-orthogonal view.
+
+MDHistoWorkspace
+~~~~~~~~~~~~~~~~
+
+1. Make a 2D and 3D MDHistoWorkspaces
```

Comment:
```suggestion
1. Make a 3D MDHistoWorkspaces
```
The code block only makes a 3D

Reply:
Ah good point  - the 2D test is covered by the `md_4D_svrebinned` stuff